authentication-typescript/template/src/navigation/AppRouter.tsx

```diff
@@ -0,0 +1,124 @@
+import { routes } from './Routing';
+import { ExampleHome } from '../screens/ExampleHome';
+import i18nAppInstance from '../translations/i18n';
+import { ChangePassword } from '../components/ChangePassword';
+
+export const AppRouter: React.FC = () => {
+    const navigate = useNavigate();
+    const app = useApp();
+    const { email, rememberMe } = app.loginData;
+
+    return (
+        <Routes>
+            {/* AUTH ROUTES */}
+            <Route
+                element={
+                    <AuthContextProvider
+                        actions={ProjectAuthUIActions(app)}
+                        language={app.language}
+                        navigate={navigate}
+                        routeConfig={routes}
+                        i18n={i18nAppInstance}
+                        rememberMeDetails={{ email: rememberMe ? email : '', rememberMe: rememberMe }}
+                    >
+                        <Outlet />
+                    </AuthContextProvider>
+                }
+            >
+                <Route
+                    path={'/login'}
+                    element={
+                        <ReactRouterGuestGuard isAuthenticated={app.isAuthenticated} fallBackUrl={'/'}>
+                            <Login />
+                        </ReactRouterGuestGuard>
+                    }
+                />
+                <Route
+                    path={'/forgot-password'}
+                    element={
+                        <ReactRouterGuestGuard isAuthenticated={app.isAuthenticated} fallBackUrl={'/'}>
+                            <ForgotPasswordScreen />
+                        </ReactRouterGuestGuard>
+                    }
+                />
+                <Route
+                    path={'/contact-support'}
+                    element={
+                        <ReactRouterGuestGuard isAuthenticated={app.isAuthenticated} fallBackUrl={'/'}>
+                            <ContactSupportScreen />
+                        </ReactRouterGuestGuard>
+                    }
+                />
+                <Route
+                    path={'/reset-password'}
+                    element={
+                        <ReactRouterGuestGuard isAuthenticated={app.isAuthenticated} fallBackUrl={'/'}>
+                            <ResetPasswordScreen />
+                        </ReactRouterGuestGuard>
+                    }
+                />
+                {/* USER APPLICATION ROUTES */}
+                <Route
+                    element={
+                        <>
+                            <Outlet />
+                            {app.showChangePasswordDialog && <ChangePassword />}
+                        </>
+                    }
+                >
+                    <Route
+                        path={'/homepage'}
+                        element={
+                            <ReactRouterAuthGuard isAuthenticated={app.isAuthenticated} fallBackUrl={'/login'}>
+                                <ExampleHome />
```

Comment:
It looks like we only have one page set up. The Authentication Template is supposed to extend from the routing template, so it should have multiple pages and a navigation drawer.